tools/witx/src/docs/ast.rs

```diff
@@ -26,7 +27,13 @@ impl ToMarkdown for Document {
                 heading.new_level_down(),
                 name,
                 name,
-                &d.docs,
+                format!(
+                    "{}Size: {}\n Alignment: {}",
```

Comment:
Is this space before the "Alignment:" intended? It seems to make the spacing inconsistent with "Size:" and "Offset:".